src/app/legacy/psammead/psammead-navigation/src/index.jsx

```diff
@@ -64,7 +63,7 @@ const ListItemBorder = `
 const StyledLink = styled.a`
   ${({ script }) => script && getPica(script)};
   ${({ service }) => getSansRegular(service)};
-  ${({ brandForegroundColour }) => `color: ${brandForegroundColour};`}
+  color: ${props => props.theme.palette.BRAND_FOREGROUND};
```

Comment:
Not sure why there are two delcaration of colour. Should the one I have changed be removed?